src/embed/search.ts

```
@@ -161,6 +162,9 @@ export class SearchEmbed extends TsEmbed {
         if (hiddenActions?.length) {
             queryParams[Param.HideActions] = hiddenActions;
         }
+        if (visibleActions?.length) {
+            queryParams[Param.VisibleActions] = visibleActions;
+        }
```

I see action handling in all 3 flavors search/app/pinboard, can we converge these to be in the single place ?

Also, lets throw an error when the user uses both `visibleActions` and `hiddenActions`

---

since now we support Actions for each embed flavour, shall we add it in Base Class.